src/main/java/com/imsweb/mph/DefaultHematoDataProvider.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2016 Information Management Services, Inc.
+ */
+package com.imsweb.mph;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvException;
+
+import com.imsweb.mph.internal.HematoDTO;
+
+/**
+ * This is a default hemato db data provider which uses seer-api to get same primary, transform to or transform from data from
+ * hematopoietic and lymphoid neoplasm database.
+ */
+@SuppressWarnings("java:S112") //This is a lab class and throwing RuntimeException is fine
```

Comment:
I didn't see this before because this class had not been modified, but this comment is wrong; this is not a lab class. And there is no reason to suppress that one when we can just replace the RuntimeException by IllegalStateException. I am not a huge fan of that, but that's what we do everywhere else. So can you please just do it here as well? Thanks.